client/app/views/partials/user.html

```diff
@@ -16,7 +16,8 @@
       <span class="text-separator">|</span>
     </span>
     <span id="SupportLinkBox" data-ng-if="Authentication.session && Authentication.session.role !== 'whistleblower'">
-      <a id="SupportLink" href="" data-ng-click="Utils.openSupportModal()" data-uib-tooltip="{{'Request support' | translate}}"><i class="fa-solid fa-fw fa-life-ring"></i></a>
+      <a data-ng-if="!public.node.custom_support_url" id="SupportLink" href="" data-ng-click="Utils.openSupportModal()" data-uib-tooltip="{{'Request support' | translate}}"><i class="fa-solid fa-fw fa-life-ring"></i></a>
+      <a data-ng-if="public.node.custom_support_url" id="SupportLink" href="{{public.node.custom_support_url}}" data-uib-tooltip="{{'Request support' | translate}}" target="_blank"><i class="fa-solid fa-fw fa-life-ring"></i></a>
```

Comment:
considering that this function Utils.openSupportModal() is called also in other views, i suggest we could implement the entire redirect logic inside the javascript function instead of inside the html view . this would reduce the code duplication